cypress/integration/WalletPage/WalletPage.ts

```diff
@@ -0,0 +1,33 @@
+import {Given, Then} from "cypress-cucumber-preprocessor/steps";
+import {LandingPage} from "../../pageObjects/pages/LandingPage";
+import {WalletPage} from "../../pageObjects/pages/WalletPage";
+import {CommonElements} from "../../pageObjects/components/CommonElements";
+
+Given(`User has opened the {string} page on {string}`, (page, network) => {
+  LandingPage.openDataPage(page, network)
+});
+
+Given(`The account address, type and network is shown correctly for {string}`, (network) => {
+  WalletPage.validateWalletAddressTypeAndNetwork(network)
```

Comment:
In the codebase, "wallet" is never mentioned. "Account" is used instead.

Reply:
Renamed all "Wallet" to "Account"
![Screenshot 2022-03-08 at 11 18 46](https://user-images.githubusercontent.com/77115130/157206410-ec313382-03ee-48f7-87ba-ceb5675370d6.png)